src/Assignments/SimpleProblemPage.tsx

```diff
@@ -529,10 +535,14 @@ export const SimpleProblemPage: React.FC<SimpleProblemPageProps> = () => {
                                 setAttemptsRemaining={setAttemptsRemaining}
                                 setOpenDrawer={_.isNil(selectedGradeId) ? undefined : setOpenDrawer}
                             />
-                            {/* Temporarily disabled for release.  */}
-                            {false && (<a href="https://openlab.citytech.cuny.edu/ol-webwork/" rel="noopener noreferrer" target="_blank" >
-                                <Button className='float-right'>Ask for help</Button>
-                            </a>)}
+                            {/* Custom feature for CityTech (university #5). TODO: make this solution more robust */}
+                            {selectedProblemId && _.find(users, ['id', getUserId()])?.universityId === 5 &&
```

Comment:
This will break canada right away...
Maybe we should have a flag that we return on login and keep in local storage
```
canAskForHelp
```
and when we log a user in we check the university name not id

Reply:
weak entities (lookup tables, enums) like topic types are ok to use the id because the same for all installations
one cleanup task we should do is add sequelize seeders